src/components/voting/voteList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './voteList.css';
+
+const VoteList = ({
+  list, title, className, votes,
+}) => (
+  <React.Fragment>
+  {list.length > 0 ?
```

Comment:
you can do in here  list.length don't need the > 0

Reply:
https://github.com/LiskHQ/lisk-hub/pull/2085/commits/920d8064b3c6e76670313b6d7b8e8100bf754956